app/src/main/java/io/islnd/android/islnd/messaging/MessageLayer.java

```diff
@@ -36,6 +37,13 @@ private static boolean addFriendToDatabaseAndCreateDefaultProfile(Context contex
         Profile profile = Util.buildDefaultProfile(context, identity.getDisplayName());
         DataUtils.insertProfile(context, profile, userId);
 
+        DataUtils.insertNotification(
+                context,
```

Comment:
I think we should hide the setting of the server time, the `NotificationType.NEW_FRIEND`, and the fact there is no postId by creating a `NewFriendNotification` class.

Then this code would look something like
`NewFriendNotification newFriendNotification = new NewFriendNotification(userId);`
`DataUtils.insertNotification(context, newFriendNotification);`

This is just an idea. I'm not sure it is the best option.